nautobot/core/apps/__init__.py

```diff
@@ -124,6 +124,11 @@ def register_homepage_panels(path, label, homepage_layout):
     define different parts of the layout.
 
     These objects are converted into a dictionary to be stored inside of the Nautobot registry.
+
+    Args:
+        path (str): Path to the app which defines the homepage layout.
+        label (str): Label of the app which defines the homepage layout.
+        homepage_layout (list): A list of objects which define the homepage layout.
```

Comment:
```suggestion
        path (str): Absolute filesystem path to the app which defines the homepage layout; typically this will be an `AppConfig.path` property
        label (str): Label of the app which defines the homepage layout, for example `dcim` or `my_nautobot_plugin`
        homepage_layout (list): A list of HomePagePanel instances which define the homepage layout.
```